src/System.Text.Json/src/System/Text/Json/Node/JsonString.cs

```diff
@@ -208,5 +209,55 @@ public string Value
         ///   Returns <see cref="JsonValueKind.String"/>
         /// </summary>
         public override JsonValueKind ValueKind { get => JsonValueKind.String; }
+
+        /// <summary>
+        ///   Converts the text value of this instance, which should encode binary data as base-64 digits, to an equivalent 8-bit unsigned <see cref="byte"/> array.
+        ///   The return value indicates wether the conversion succeeded.
+        /// </summary>
+        /// <param name="value">
+        ///   When this method returns, contains the <see cref="byte"/> array equivalent of the text contained in this instance,
+        ///   if the conversion succeeded.
+        /// </param>
+        /// <returns>
+        ///   <see langword="true"/> if text was converted successfully; othwerwise returns <see langword="false"/>.
+        /// </returns>
+        internal bool TryGetBytesFromBase64(out byte[] value)
+        {
+            if (_value.Length < 1)
```

Comment:
```suggestion
            if (_value.Length < 4)
```
The minimum valid base64 string must be of length 4*. So you could short-circuit the method. Otherwise it has to try to decode and this will return `false` -- hence no need to go so far.

\* 1 byte data base64 encodes to 2 bytes + 2 padding-bytes = 4 bytes = 4 ASCII-chars

Reply:
Completely forgot about this, you're right. I'll commit your suggestion.